src/pages/Data/Data.tsx

```
@@ -1,78 +1,98 @@
+  const perPageParams = [5, 10, 15];
 
   useEffect(() => {
     dispatch(fetchOrders());
   }, [dispatch]);
 
-  const toggleAdditionalCols = () => dispatch(toggleCols());
-
   const showSelectedPage = (index: number) => dispatch(showPage(index));
 
   const showOrderInfo = (orderInfo: OrderInfo) =>
     dispatch(showOrder(orderInfo));
 
   const hideOrderInfo = () => dispatch(hideOrder());
 
-  const filtrateOrdersByStatus = (status: number) =>
-    dispatch(filterByStatus(status));
-
-  const cancelFiltrateOrdersByStatus = () => dispatch(cancelFilterByStatus());
-
-  const state: IDataState = useSelector((root: RootType) => root.orders);
-
   const paginationSlice = (orders: Array<OrderInfo>): Array<OrderInfo> =>
     orders.slice(
-      state.paginationShift * itemsCountPerPage,
-      state.paginationShift * itemsCountPerPage + itemsCountPerPage
+      state.paginationShift * state.itemsCountPerPage,
+      state.paginationShift * state.itemsCountPerPage + state.itemsCountPerPage
     );
 
+  const changePerPageCount = (count: number) =>
+    dispatch(changePaginationCount(count));
+
+  const toggleHead = (item: HeaderType) => dispatch(toggleAdditionalCols(item));
+
+  const addTranslation = (items: Array<HeaderType>): Array<HeaderType> => {
+    return items.map((item) => {
+      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+      return { ...item, title: t(item.title!) };
+    });
+  };
+
   return (
     <div className={`data ${className}`}>
-      <FilterPanel
-        onItemClick={filtrateOrdersByStatus}
-        onClearClick={cancelFiltrateOrdersByStatus}
-        statuses={processStatuses}
-      />
       {state.orderForShow && (
         <Order orderInfo={state.orderForShow} hideOrder={hideOrderInfo} />
       )}
+
+      <div className="data__dashboard">
+        <Dropdown
+          items={headerOptionalColumns}
+          translatedItems={addTranslation(headerOptionalColumns)}
```

should contains all columns. Default columns should be visible by default but user should be able to hide them. Optional columns are hidden by default but user should be able to show them. Also it would be good if you add a button to reset this settings to default